### ISSUE TITLE
Plane- Delete

### ISSUE THREAD

**Issue by rarceneaux** [nss-evening-cohort-10/nutshell-panam-airlines#21]
# User Story

- As a user, I should be able to delete a plane on the Fleet Inventory/Main dashboard when I click the delete button of the selected plane card.

# Dev Notes

- Create `ra-deletePlane` branch
- Create `axios.delete()` in planeData.js file
- Create deletePlane function in plane.js file
-  Create delete button/event listener in the plane.js file 
<img width="717" alt="Screen Shot 2019-11-14 at 9 30 45 PM" src="https://user-images.githubusercontent.com/40898992/68916953-d2d02300-072d-11ea-9556-7e8046c890b0.png">
- The event will delete the plane object from firebase

# AC
**WHEN** a user clicks on the delete button on the selected plane card the plane card will disappear from the page
**AND** the selected plane object will be removed from firebase

**Comment by rarceneaux** [comment]
<img width="485" alt="Screen Shot 2019-11-13 at 11 53 23 AM" src="https://user-images.githubusercontent.com/40898992/68797722-ba231880-061a-11ea-9090-515eac638ee9.png">

**Comment by michaelclark2** [comment]
I would reword the AC to read with a WHEN THEN AND statement and include more info about where you are going to write the event listener that will be attached to the delete buttons
